src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Activity Feed</title>
+    <link rel="stylesheet" href="/css/main.css">
+</head>
+
+<body>
+
+    <nav>
+        <a id="navTitle" href="/">CodeU Chat App</a>
+        <a href="/conversations">Conversations</a>
+        <% if(request.getSession().getAttribute("user") != null) { %>
```

Comment:
The indentation is a little bit weird, but maybe is the Github editor.

Reply:
I think it was the auto-indentation of my text editor, will fix.

Reply:
Resolved.

Reply:
Resolved.